controllers/masteruserrecord/sync.go

```diff
@@ -195,9 +196,18 @@ func (s *Synchronizer) alignReadiness() (bool, error) {
 		// if there is no existing notification with these labels
 		if len(notificationList.Items) == 0 {
 
+			config, err := toolchainconfig.GetToolchainConfig(s.hostClient)
+			if err != nil {
+				return false, errs.Wrapf(err, "unable to get ToolchainConfig")
+			}
+
+			keysAndVals := map[string]string{
+				"RegistrationURL": config.RegistrationService().RegistrationServiceURL(),
```

Comment:
Let's use constants for notification keys.

Reply:
Done 39b6f7c058b917f01984d2f7b3268aa562d68e40